deb/openmediavault/workbench/src/app/shared/services/constraint.service.ts

```diff
@@ -102,8 +102,10 @@ export class ConstraintService {
       let result;
       if (_.isPlainObject(node)) {
         if (_.has(node, 'prop')) {
+          const containsToken =
```

Comment:
Is there a unit test for that. I would like to have an example where it is necessary to tokenize the property field.

Reply:
IMO tokeinzed strings should be formatted at an earlier stage, e.g. https://github.com/openmediavault/openmediavault/commit/ce5d7b88f3a6657798161ffa56500885bfab71da